src/demo-app/app/shared/media-query-status.ts

```diff
@@ -31,7 +31,9 @@ export class MediaQueryStatus implements OnDestroy {
   private _watcher : Subscription;
   activeMediaQuery : string;
 
-  constructor(media$ : ObservableMedia) { this.watchMediaQueries(media$); }
+  constructor(media$ : ObservableMedia) {
+    this.watchMediaQueries(media$);
```

Comment:
this is a community standard to indicate a variable the references an observable. I believe this is used consistently throughout the library.